test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -327,6 +328,28 @@ var _ = SIGDescribe("GracefulNodeShutdown [Serial] [NodeFeature:GracefulNodeShut
 				podStatusUpdateTimeout+(nodeShutdownGracePeriod-nodeShutdownGracePeriodCriticalPods),
 				pollInterval).Should(gomega.Succeed())
 
+			ginkgo.By("Verify that all pod ready to start condition are set to false after terminating")
+			// All pod ready to start condition should set to false
+			gomega.Eventually(ctx, func(ctx context.Context) error {
+				list, err = e2epod.NewPodClient(f).List(ctx, metav1.ListOptions{
+					FieldSelector: nodeSelector,
+				})
+				if err != nil {
+					return err
+				}
+				framework.ExpectEqual(len(list.Items), len(pods), "the number of pods are not as expected")
+
```

Comment:
this is why linter hints fails, should we replace? if so, I think the rest of the code should update as well
<img width="1137" alt="image" src="https://github.com/kubernetes/kubernetes/assets/73322208/87d3f08f-f0a8-4898-98b3-2ad2c50bf540">

Reply:
You can replace your code.  And maybe open up a new PR switching over if you want for the others

Reply:
updated